### ISSUE TITLE
Alt text missing

### ISSUE THREAD

**Issue by Cheesycats** [NHSDigital/data-viz-community-of-practice#3]
I’m not sure your images have alt text? I’ve tested them with a screen reader and for example the image below just reads as 1, 2, 3 so it isn’t meaningful to a user with a screen reader.

![image](https://github.com/user-attachments/assets/25ac7456-7522-474f-ae22-3b66c7f5d941)

The image for NHS logo guidance just states something like 'NHS logo link' so individuals with a screen reader get nothing useful from that image

![image](https://github.com/user-attachments/assets/0d187e31-1ef9-40c7-b91a-4501e269e91e)
